UI/MainMenu/MainMenu.tscn

```diff
@@ -1,524 +1,551 @@
+[ext_resource path="res://Game/TooltipDisplay.tscn" type="PackedScene" id=7]
+[ext_resource path="res://Events/EventSystem.gd" type="Script" id=8]
+[ext_resource path="res://Quests/QuestSystem.gd" type="Script" id=9]
+[ext_resource path="res://Game/Pregnancy/ChildSystem.gd" type="Script" id=10]
+[ext_resource path="res://UI/CharactersArtworkPanel/CharactersArtworkPanel.tscn" type="PackedScene" id=11]
+[ext_resource path="res://Game/PlayerPanel.tscn" type="PackedScene" id=12]
+[ext_resource path="res://Game/CharacterPanel.tscn" type="PackedScene" id=13]
+[ext_resource path="res://UI/CharactersPanel.tscn" type="PackedScene" id=14]
+[ext_resource path="res://UI/MainMenu/InGameMenu.tscn" type="PackedScene" id=15]
+[ext_resource path="res://UI/SkillsUI/SkillsUI.tscn" type="PackedScene" id=16]
+[ext_resource path="res://UI/DebugUI/DebugPanel.tscn" type="PackedScene" id=17]
+[ext_resource path="res://Game/MainScene.gd" type="Script" id=18]
+[ext_resource path="res://UI/Options/OptionsScreen.tscn" type="PackedScene" id=19]
+[ext_resource path="res://UI/MainMenu/CreditsScreen.tscn" type="PackedScene" id=20]
+[ext_resource path="res://UI/ModsMenu/ModsMenu.tscn" type="PackedScene" id=21]
+[ext_resource path="res://UI/MainMenu/LoadGameScreen.tscn" type="PackedScene" id=22]
+[ext_resource path="res://Util/SceneConverter.tscn" type="PackedScene" id=23]
+[ext_resource path="res://UI/LikesGenerator/NpcLikesGenerator.tscn" type="PackedScene" id=24]
+[ext_resource path="res://UI/QuickCharacterCreator/QuickCharacterCreator.tscn" type="PackedScene" id=25]
+[ext_resource path="res://UI/ModMaker/ModMaker.tscn" type="PackedScene" id=26]
+[ext_resource path="res://Game/SmartScrollContainer.gd" type="Script" id=27]
+
+[sub_resource type="StyleBoxFlat" id=2]
 bg_color = Color( 0.207843, 0.133333, 0.364706, 1 )
 corner_radius_top_right = 16
 corner_radius_bottom_right = 16
 
-[sub_resource type="DynamicFont" id=6]
+[sub_resource type="DynamicFont" id=9]
 size = 30
-font_data = ExtResource( 7 )
-
-[sub_resource type="DynamicFont" id=3]
-size = 38
-outline_size = 2
-outline_color = Color( 0, 0, 0, 1 )
-font_data = ExtResource( 2 )
-
-[sub_resource type="StyleBoxFlat" id=4]
-bg_color = Color( 0.364706, 0.133333, 0.329412, 1 )
-border_width_left = 4
-border_width_top = 4
-border_width_right = 4
-border_width_bottom = 4
-border_color = Color( 0.364706, 0.133333, 0.329412, 1 )
-corner_radius_top_left = 32
-corner_radius_top_right = 32
-corner_radius_bottom_right = 32
-corner_radius_bottom_left = 32
-
-[sub_resource type="DynamicFont" id=5]
-size = 24
-font_data = ExtResource( 2 )
-
-[sub_resource type="StyleBoxFlat" id=2]
+font_data = ExtResource( 5 )
+
+[sub_resource type="GDScript" id=8]
+script/source = "extends RichTextLabel
+
+func _ready():
+	var _ok = GlobalRegistry.connect(\"donationDataUpdated\", self, \"updateDonationData\")
+	updateDonationData()
+	
+func updateDonationData():
+	bbcode_text = GlobalRegistry.getDonationDataString()
```

Comment:
Don't do embedded scripts, they are hard to review and make the project messy. Code should only be in .gd files

Reply:
Ok. I'll keep in mind.